new-integration-tests/src/test/java/oracle/weblogic/kubernetes/TestConstants.java

```diff
@@ -90,6 +90,21 @@
   public static final String VOYAGER_CHART_NAME = "voyager";
   public static final String VOYAGER_CHART_VERSION = "12.0.0";
 
+  // ELK Stack and WebLogic logging exporter constants
+  public static final String ELASTICSEARCH_NAME = "elasticsearch";
+  public static final String ELASTICSEARCH_IMAGE = ELASTICSEARCH_NAME + ":6.8.0";
+  public static final String ELASTICSEARCH_HOST = "elasticsearch.default.svc.cluster.local";
+  public static final int ELASTICSEARCH_HTTP_PORT = 9200;
+  public static final int ELASTICSEARCH_HTTPS_PORT = 9300;
+  public static final String ELKSTACK_NAMESPACE = "default";
+  public static final String LOGSTASH_INDEX_KEY = "logstash";
+  public static final String WEBLOGIC_INDEX_KEY = "wls";
+  public static final String KIBANA_INDEX_KEY = "kibana";
+  public static final String KIBANA_NAME = "kibana";
+  public static final String KIBANA_IMAGE = KIBANA_NAME + ":6.8.0";
```

Comment:
Can we include the image tag in the String KIBANA_IMAGE  instead of hard-coding the tag  (6.8.0)

Reply:
fixed